README.md

```diff
@@ -1,25 +1,78 @@
 
-In order to deploy on Docker, you will need to provide the `PLAY_SECRET_KEY`.  The service will throw an exception and quit at startup unless you do.  You can generate a new secret using `sbt playGenerateSecret` or from some other means such as using a static value or generating one from other tooling.  If you don't already have an sbt server running, using something like `head -c 30 /dev/random | base64` may produce a value more quickly.
+Create a `prod.env` file containing all configuration outlined above. Then publish your project locally and run the the container.
+
+```
+sbt docker:publishLocal
+
+docker run -p 8000:9000 -e PLAY_SECRET_KEY="$(head -c 30 /dev/random | base64)" --env-file prod.env catalyst-slack-service:1.0-SNAPSHOT
+```
+
+In the above example, we expose the host/local port 8000 to the container's port 9000 (the default port for the Play framework).  We've also chosen to generate a random secret key and use the default Docker tag.  If we had specified an environment variable during the build phase (`VERSION_NUMBER=example sbt docker:publishLocal`), then our tag would be `catalyst-slack-service:example`.
+
+## Configure Slack App
+
+After getting your local service running you will need to use [ngrok](https://dashboard.ngrok.com/get-started) to expose your local server. 
+
+Setup and connect your ngrok account then expose your local service.
+
+`./ngrok http 4542`
 
-Create a `prod.env` file containing all the slack environment variables for the service.
+Take note of the forwarding https url. Next you need to update your Slack App with this forwarding url. 
 
-* `sbt docker:publishLocal`
-* `docker run -p 8000:9000 -e PLAY_SECRET_KEY="$(head -c 30 /dev/random | base64)" --env-file prod.env catalyst-slack-service:1.0-SNAPSHOT`
+Navigate to api.slack.com/apps.
 
-In the above example, we expose the host/local port 8000 to the container's port 9000 (the default port for the Play framework).  We've also chosen to generate a random secret key and use the default Docker tag.  If we had specified an environment variable during the build phase (`VERSION_NUMBER=example sbt docker:publishLocal`), then our tag would be `catalyst-slack-service:example`.
\ No newline at end of file
+1. Update **Interactive Components** with the **Request URL** `{FORWARDING_URL}/bias-correct/v2/slack/actions`.
+2. Update **OAuth & Permissions** with the **Redirect URL** `{FORWARDING_URL}/bias-correct/v2/slack/auth/redirect`.
+3. Update **Event Subscriptions** with the **Request URL** `{FORWARDING_URL}/bias-correct/v2/slack/events`.
```

Comment:
looks great. let's add 4. Update ** Slash Commands** with the **Request URL** `{FORWARDING_URL}/bias-correct/v2/slack/help`.